finagle-core/src/main/scala/com/twitter/finagle/netty3/Netty3Transporter.scala

```diff
@@ -139,7 +141,7 @@ object Netty3Transporter {
       pipelineFactory,
       newChannel = cf.newChannel(_),
       newTransport = (ch: Channel) => newTransport(ch).cast[In, Out],
-      tlsConfig = tls map { case engine => Netty3TransporterTLSConfig(engine, tlsHostname) },
+      tlsConfig = tls map { case engine => Netty3TransporterTLSConfig(engine, tlsHostname, sslHandshakeTimeout) },
```

Comment:
I know you didn't add it but the 'case' keyword isn't needed so do you mind cleaning it up while you're here?